oaknational/Oak-Web-Application
# Fix/pupil 815/add alt text quiz images

## Description

Music year: 1988

- List of changes
images on quiz have generic alt text 

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2961--oak-web-application.netlify.thenational.academy/pupils/lessons/identifying-perpendicular-linear-graphs/starter-quiz?share=true
3. You should see images with alt text

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
